sympy/series/tests/test_limits.py

```diff
@@ -880,6 +880,14 @@ def test_issue_18118():
     assert limit(sign(sin(x)), x, 0, "-") == -1
     assert limit(sign(sin(x)), x, 0, "+") == 1
 
+def test_issue_18176():
+    x = Symbol('x', real=True , positive= True)
+    n = Symbol('n', integer=True, positive=True)
+    k = 1
+    assert limit(x**n-x**(n-k), x, oo) == oo
```

Comment:
`k` can directly be substituted here without a separate line.

Reply:
Yeah sure , I'll do the needful . I just thought that the issue was reported using k so I should go forward with using k explicitly !